go/cmd/betabot/main.go

```diff
@@ -324,6 +324,8 @@ func (bot *Bot) handleEvent(ctx context.Context, gme *bertymessenger.EventStream
 					time.Sleep(1 * time.Second)
 					{
 						// create staff conversation
+						displayName := fmt.Sprintf(staffXConvPrefix + conv.ContactDisplayName)
```

Comment:
```suggestion
// Upper in the code base
staffXConvPrefix = "Berty Staff X %s"
// Here
						displayName := fmt.Sprintf(staffXConvPrefix, conv.ContactDisplayName)
```
**OR** (I like more this solution personaly):
```suggestion
						displayName := staffXConvPrefix + conv.ContactDisplayName
```